trunk/NDHMS/Routing/module_NWM_io_dict.F

```diff
@@ -168,7 +168,7 @@ module module_NWM_io_dict
    character(len=1024), dimension(20) :: crsCharAttNames,crsFloatAttNames
    character(len=1024), dimension(20) :: crsCharAttVals
    real, dimension(20,20) :: crsRealAttVals
-   real, dimension(20) :: crsRealAttLen
+   integer, dimension(20) :: crsRealAttLen
```

Comment:
Should the variable be renamed? Why was it a real in the first place? The other option here is to promote tmpLen to a real.
ldasOutDict%xRealAttLen(xRealAttCnt+1) = real(tmpLen)